File: Exercises/Foundational/Intermediate-Level/3/java/E_O.java
import java.util.Scanner;
class E_O{
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number: ");
        Float num = scanner.nextFloat();
        String result = (num % 2 == 0) ? "even" : "odd";
        System.out.println(num + " is " + result);
        scanner.close();
    }
}

File: Exercises/Foundational/Intermediate-Level/3/js/E_O.js
const { createInterface } = require("readline/promises");
(async () => {
  const input = createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  const num = await input.question("Enter a number: ");
  const result = num % 2 == 0 ? "even" : "odd";
  console.log(`${num} is ${result}`);
  input.close();
})();
